src/js/form-builder.js

```diff
@@ -686,7 +686,7 @@ function FormBuilder(opts, element, $) {
         } else if (attrData.hasOwnProperty('value') || attrData.hasOwnProperty('checked')) {
           isChecked = attrData.value || attrData.checked || false
         }
-        return boolAttribute(attr, { ...attrData, [attr]: isChecked }, { first: attrData.label })
+        return boolAttribute(attr, { ...attrData, [attr]: isChecked }, { first: i18n[attr] })
```

Comment:
would something like this work?
```suggestion
        return boolAttribute(attr, { ...attrData, [attr]: isChecked }, { first: attrData.label || i18n[attr] })
```

Reply:
i18n[attr] is calculated from attrData.label.

The following Jest testcase fails if you use attrData.label in

```javascript
describe('FormBuilder stage names translated', () => {
  test('boolAttribute name translated', async () => {
    const config = {
      typeUserAttrs: {
        text: {
          readonly: {
            label: ["readOnly"],
            value: false,
            type: 'checkbox',
          }
        }
      },
      i18n: {
        override: {
          "en-US": {
            readOnly: "Is Readonly",
          }
        }
      }
    }

    const fbWrap = $('<div>')
    const fb = await $(fbWrap).formBuilder(config).promise
    const field = {
      type: 'text',
      class: 'form-control'
    }
    fb.actions.addField(field)
    expect(fbWrap.find('.readonly-wrap label').text()).toBe(config.i18n.override['en-US'].readOnly)
  })
})
```

With the suggested change the test case fails because the label is untranslated:

```
    Expected: "Is Readonly"
    Received: "readOnly"
```

Reply:
i see, thanks for explaining